.github/workflows/pr.yaml

```diff
@@ -0,0 +1,93 @@
+name: PR checks
+on: pull_request
+env:
+  SECRET_KEY: insecure_dev_key
```

Comment:
Which part needs this? `docker build` needs the variable passed explicitly, and `dotrun` should get it from `.env`. In fact, that should be part of the point of that test - to check it runs without needing extra environment variables.

Is it the tests maybe that need this? Could I ask you to move this into the individual jobs that need it instead (if indeed they do need it), like `SECRET_KEY=insecure_dev_key coverage run  --source=. -m unittest discover tests`.